server/core/services.py

```
@@ -84,4 +86,134 @@ def logout_user(request):
+                club.links.add(link_obj)
+
+            tags = data.get('tags', [])
+            for tag_name in tags:
+                tag_obj, created = Tag.objects.get_or_create(tag=tag_name)
+                club.tags.add(tag_obj)
+
+            club.save()
+            return club
+        
+        except ValidationError as e:
+                raise ValidationError({'project': e.messages})
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)